slr/temp/urls.py

```diff
@@ -0,0 +1,16 @@
+from django.urls import path
+
+from . import views
+from django.urls import path
+
+app_name = 'temp'
+
+urlpatterns = [
+    path('', views.IndexView.as_view(), name='index'),
+
+]
+'''
+    path('frontend/', views.FrontEndView.as_view(), name='frontend'),
+    path('backend/', views.BackEndView.as_view(), name='backend'),
+    path('frontendsubmission/', views.FrontEndSubmissionView.as_view(), name='frontend_submission'),
+    '''
```

Comment:
You could've deleted this, there's no need to have these lines in the application when they're already present in the main application.